examples/LaserWakefield/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -102,6 +102,27 @@ typedef Particles<ParticleDescription<
 
 /*--------------------------- ions -------------------------------------------*/
 
+/*! Specify (chemical) element
+ * 
+ * Proton and neutron numbers define the chemical element that the ion species
+ * is based on. This value can be non-integer for physical models taking  
+ * charge shielding effects into account. @see http://en.wikipedia.org/wiki/Effective_nuclear_charge
+ * 
+ * It is wrapped into a struct because of C++ restricting floats from being 
+ * template arguments. 
+ * 
+ * Do not forget to set the correct mass of the atom in @see physicalConstants.param !
+ */
+struct NumberOfProtons
+{
+    static const float_X value = 2.0;
+};
+
+struct NumberOfNeutrons
```

Comment:
`coreElements` is only a example. But we need a fixed name that all algorithms can access this flags to calculate mass and/or charge.
`coreElements` is only an alias like `pusher`,`ionizer`,... and must defined in `speciesAttributes.param`. It can not changed by the user and must well named by the developer of PIConGPU.